src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -523,30 +553,95 @@ export default class ReviewConfirmDissolution extends Mixins(DateMixin) {
   }
 }
 
-::v-deep #certify-section, ::v-deep #document-delivery-section {
+// styles common to the sections
```

Comment:
These overrides are here because they can't be in the sub-components (some of which are external). Or they aren't in the sub-components so that we can style them all the same here.